packages/story-editor/src/components/library/panes/text/textSets/textSets.js

```diff
@@ -81,7 +81,7 @@ function TextSets({ paneRef, filteredTextSets }) {
           columnWidth={TEXT_SET_SIZE}
           rowHeight={TEXT_SET_SIZE}
           onFocus={handleGridFocus}
-          role="list"
+          role="group"
```

Comment:
`list` necessitates `listitem`, what we have is a group of buttons so updating to have corresponding role.